### PR TITLE
Added support for project organization policy

### PR DESCRIPTION
**Pull request by rdayama-eng**
Corresponding magic module PR

### CODE REVIEW

**Comment by rdayama-eng** [comment]
@morgante Please take a look

File: converters/google/convert.go [hunk 1]
```diff
@@ -223,12 +267,51 @@ func (c *Converter) augmentAsset(tfData converter.TerraformResourceData, cfg *co
...
+			if o.BooleanPolicy != nil {
+				booleanPolicy = &BooleanPolicy{
+					Enforced: o.BooleanPolicy.Enforced,
+				}
+			}
+			if o.RestoreDefault != nil {
+				restoreDefault = &RestoreDefault{}
+			}
+			currentTime := time.Now()
+			currentTime = currentTime.Round(time.Minute)
```
Comment (morgante) [inline converters/google/convert.go:294]: Should add a comment about why we're rounding.

File: testdata/templates/example_project_organization_policy.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  version     = "~> {{.Provider.version}}"
+  {{if .Provider.credentials }}credentials = "{{.Provider.credentials}}"{{end}}
+}
+
+resource "google_project_organization_policy" "serial_port_policy" {
```
Comment (morgante) [inline testdata/templates/example_project_organization_policy.tf:22]: We should add list and default cases as well.

File: converters/google/convert.go (hunk 1 above)
Comment (rdayama-eng) [reply converters/google/convert.go:294]: Added a comment

File: testdata/templates/example_project_organization_policy.tf (hunk 2 above)
Comment (rdayama-eng) [reply testdata/templates/example_project_organization_policy.tf:22]: Updated test accordingly and added merging logic as well. Raised a corresponding PR https://github.com/GoogleCloudPlatform/magic-modules/pull/4250

File: converters/google/convert.go (hunk 1 above)
Comment (morgante) [reply converters/google/convert.go:294]: I don't see a comment?

Comment (rdayama-eng) [reply converters/google/convert.go:295]: Actually I added comment in test file, added comment here as well

**Review by morgante — APPROVED** [review]

**Comment by morgante** [comment]
Note: this change includes some cherry-picked fixes in vendor/, we should resolve https://github.com/GoogleCloudPlatform/terraform-validator/issues/158 soon after.

**Pull request merged**